util/CosmosHelper.js

```diff
@@ -190,29 +216,11 @@ async function MsgSendMultiple(
       outputs,
     },
   };
+  const broadcastedTx = await signingCosmosClient.signAndBroadcast([sendMsg], fee, memo);
   return {
-    message,
-    simulate: ({ memo = undefined }) => {
-      let base = 30000;
-      const numberOfReceivers = message.value.outputs.length;
-      if (numberOfReceivers) base += numberOfReceivers * 8000;
-      if (memo && memo.length) base += memo.length * 100;
-      return Math.floor(base * 1.2);
-    },
-    send: (
-      { gas, gasPrices = DEFAULT_GAS_PRICE, memo = undefined },
-      signer,
-    ) => api.send(senderAddress, { gas, gasPrices, memo }, message, signer),
+    txHash: broadcastedTx.transactionHash,
+    gas: fee.gas,
+    gasPrices: fee.amount,
```

Comment:
Since `gasPrice` is changable in Keplr, should this `gasPrice` be a result from signed transaction, instead of hardcode fee?